fastlane/Constants/Constant.swift

```diff
@@ -33,6 +33,7 @@ enum Constant {
     static let outputPath = "./Output"
     static let buildPath = "\(outputPath)/Build"
     static let derivedDataPath = "\(outputPath)/DerivedData"
+    static let xcovOutputDirectoryPath: String = "./fastlane/xcov_output"
```

Comment:
Could we move `xcov_output` to `Output`?

Reply:
Yeah, I update at https://github.com/nimblehq/ios-templates/pull/366/commits/e8eef877f1d68b1bc3f281aa0b3077a7d199178e